controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -7,33 +8,49 @@
 from referee.referee import RCJSoccerReferee
 from recorder.recorder import VideoRecordAssistant
 
-automatic_mode = False
-
-TEAM_YELLOW = "The Yellows"
-TEAM_BLUE = "The Blues"
-
 
 def output_path(
     directory: Path,
-    team_blue: str,
-    team_yellow: str,
+    team_blue_id: str,
+    team_yellow_id: str,
+    match_id: int,
+    half_id: int,
 ) -> PosixPath:
 
     now_str = datetime.utcnow().strftime('%Y%m%dT%H%M%S.%fZ')
-    team_blue = team_blue.replace(' ', '_')
-    team_yellow = team_yellow.replace(' ', '_')
+    team_blue = team_blue_id.replace(' ', '_')
+    team_yellow = team_yellow_id.replace(' ', '_')
 
-    # Ensure the directory ecists
+    # Ensure the directory exists
     if not directory.exists():
         directory.mkdir(parents=True, exist_ok=True)
 
-    p = directory / Path(f'{team_blue}_vs_{team_yellow}-{now_str}')
-    return p
-
-
-output_prefix = output_path(Path('reflog'), TEAM_BLUE, TEAM_YELLOW)
+    filename = Path(f'{match_id}_-_{half_id}_-_{team_blue}_vs_{team_yellow}-{now_str}')
+    return directory / filename
+
+
+TEAM_YELLOW = os.environ.get("TEAM_YELLOW_NAME", "The Yellows")
+TEAM_YELLOW_ID = os.environ.get("TEAM_YELLOW_ID", "The Yellows")
+TEAM_YELLOW_INITIAL_SCORE = int(os.environ.get("TEAM_Y_INITIAL_SCORE", "0") or "0")
+TEAM_BLUE = os.environ.get("TEAM_BLUE_NAME", "The Blues")
+TEAM_BLUE_ID = os.environ.get("TEAM_BLUE_ID", "The Blues")
+TEAM_BLUE_INITIAL_SCORE = int(os.environ.get("TEAM_B_INITIAL_SCORE", "0") or "0")
+MATCH_ID = os.environ.get("MATCH_ID", 1)
+HALF_ID = os.environ.get("HALF_ID", 1)
+REC_FORMATS = os.environ.get("REC_FORMATS","").split(",")
+
+automatic_mode = True if "RCJ_SIM_AUTO_MODE" in os.environ.keys() else False
+
+directory = Path('/out/') if automatic_mode else Path('reflog')
+output_prefix = output_path(
+    directory,
+    TEAM_BLUE_ID,
+    TEAM_YELLOW_ID,
+    MATCH_ID,
+    HALF_ID,
+)
 reflog_path = output_prefix.with_suffix('.jsonl')
-video_path = output_prefix.with_suffix('.mp4')
+video_path = output_prefix.with_suffix('') # recorder sets suffix depending on filetype
```

Comment:
I do not like this, because `with_suffix('')` does nothing. I would instead create another method, similar to `start_recording`, but this time call it like `start_animation_recording`.